### PR TITLE
feat(dedicated.nutanix): make nutanix package clickable link

### PR DESCRIPTION
**Pull request by dectotam**
| Question         | Answer
| ---------------- | ---
| Branch?          | master
| Bug fix?         | no
| New feature?     | no
| Breaking change? | no
| Tickets          | Feat MANAGER-12973
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [ ] ~~Only FR translations have been updated~~
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~

## Description

Make "Type of pack" a link clickable in the cluster general information page

### CODE REVIEW

File: packages/manager/modules/nutanix/src/dashboard/general-info/controller.js [hunk 1]
```diff
@@ -22,6 +23,7 @@ export default class NutanixGeneralInfoCtrl {
     this.atInternet = atInternet;
     this.TRAVAUX_LINK =
       TRAVAUX_LINK[coreConfig.getRegion()] || TRAVAUX_LINK.DEFAULT;
+    this.coreConfig = coreConfig;
```
Comment (darsene) [inline packages/manager/modules/nutanix/src/dashboard/general-info/controller.js:26]:
```suggestion
    this.NUTANIX_LINK = GUIDE_PACKAGES_URL[this.coreConfig.getUser().ovhSubsidiary] ||
      GUIDE_PACKAGES_URL.DEFAULT;
```
to be compliant with the other links

File: packages/manager/modules/nutanix/src/dashboard/general-info/controller.js [hunk 2]
```diff
@@ -75,6 +77,13 @@ export default class NutanixGeneralInfoCtrl {
     }
   }
 
+  getNutanixGuideUrl() {
+    return (
+      GUIDE_PACKAGES_URL[this.coreConfig.getUser().ovhSubsidiary] ||
+      GUIDE_PACKAGES_URL.DEFAULT
+    );
+  }
```
Comment (darsene) [inline packages/manager/modules/nutanix/src/dashboard/general-info/controller.js:85]:
```suggestion
```

File: packages/manager/modules/nutanix/src/dashboard/general-info/template.html [hunk 3]
```diff
@@ -100,7 +100,18 @@
                 data-ng-if="$ctrl.isPackTypeAvailable"
             >
                 <oui-tile-description>
-                    <span data-ng-bind="$ctrl.typeOfPack"></span>
+                    <a
+                        class="oui-link oui-link_icon"
+                        data-ng-href="{{:: $ctrl.getNutanixGuideUrl() }}"
```
Comment (darsene) [inline packages/manager/modules/nutanix/src/dashboard/general-info/template.html:105]:
```suggestion
                        data-ng-href="{{:: $ctrl.NUTANIX_LINK }}"
```

**Review by darsene — CHANGES_REQUESTED** [review]

**Review by darsene — APPROVED** [review]

**Review by antonymarion — DISMISSED** [review]

**Review by JacquesLarique — DISMISSED** [review]

File: packages/manager/modules/nutanix/src/dashboard/general-info/constants.js [hunk 4]
```diff
@@ -3,6 +3,26 @@ export const TRAVAUX_LINK = {
...
+  IT: 'https://www.ovhcloud.com/it/hosted-private-cloud/nutanix/packaged',
+  NL: 'https://www.ovhcloud.com/nl/hosted-private-cloud/nutanix/packaged',
+  PL: 'https://www.ovhcloud.com/pl/hosted-private-cloud/nutanix/packaged',
+  PT: 'https://www.ovhcloud.com/pt/hosted-private-cloud/nutanix/packaged',
+  MA: 'https://www.ovhcloud.com/fr-ma/hosted-private-cloud/nutanix/packaged',
+  SN: 'https://www.ovhcloud.com/fr-sn/hosted-private-cloud/nutanix/packaged',
+  TN: 'https://www.ovhcloud.com/fr-tn/hosted-private-cloud/nutanix/packaged',
+  WS: 'https://www.ovhcloud.com/es/hosted-private-cloud/nutanix/packaged',
+  US: 'https://us.ovhcloud.com/hosted-private-cloud/nutanix/packaged',
+  DEFAULT: 'https://www.ovhcloud.com/en/hosted-private-cloud/nutanix/packaged',
```
Comment (anooparveti) [inline packages/manager/modules/nutanix/src/dashboard/general-info/constants.js:23]: **nitpick:** This is not a valid link. Maybe you can use `https://www.ovhcloud.com/en-gb/hosted-private-cloud/nutanix/packaged`

**Review by anooparveti — CHANGES_REQUESTED** [review]

**Review by antonymarion — DISMISSED** [review]

**Review by anooparveti — APPROVED** [review]

**Pull request merged**